multiversx_sdk_cli/cli_env.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import os
+from typing import Any
+
+from multiversx_sdk_cli import cli_shared
+from multiversx_sdk_cli.env import (
+    create_new_env,
+    delete_env,
+    delete_value,
+    get_active_env,
+    get_defaults,
+    get_value,
+    read_env_file,
+    resolve_env_path,
+    set_active,
+    set_value,
+)
+from multiversx_sdk_cli.utils import dump_out_json
+from multiversx_sdk_cli.ux import confirm_continuation
+
+logger = logging.getLogger("cli.env")
+
+
+def setup_parser(subparsers: Any) -> Any:
+    parser = cli_shared.add_group_subparser(
+        subparsers, "env", "Configure MultiversX CLI to use specific environment values."
+    )
+    subparsers = parser.add_subparsers()
+
+    sub = cli_shared.add_command_subparser(
+        subparsers, "env", "new", "Creates a new environment and sets it as the active environment."
+    )
+    _add_name_arg(sub)
+    sub.add_argument(
+        "--template",
+        required=False,
+        help="an environment from which to create the new environment",
+    )
+    sub.set_defaults(func=new_env)
+
+    sub = cli_shared.add_command_subparser(subparsers, "env", "get", "Gets an env value from the active environment.")
```

Comment:
Maybe `get-value` instead of `get` and `set-value` instead of `set`? Just an opinion.